### PR TITLE
Poc config

### PR DESCRIPTION
**Pull request by whosshell**

### CODE REVIEW

**Comment by whosshell** [comment]
@benj56 take a look at src/dependencies.ts file and one of the test files. Everything else is repeat of the pattern.

File: src/login/loginEpic.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```
Comment (benj56) [inline src/login/loginEpic.ts:1]: 🚔

File: src/login/loginEpic.ts [hunk 2]
```diff
@@ -61,10 +62,12 @@ const rehydrateEpic = (action$: Observable<Action>, state$: StateObservable<Root
   return action$.pipe(
     filter((action) => action.type === 'persist/REHYDRATE'),
     tap((_action) => {
-      if ((state$.value.core.login.account?.length ?? 0) > 0) {
-        timeTacApi.setConfig({ account: state$.value.core.login.account });
+      if (state$.value.core.login.account !== undefined && state$.value.core.login.account.length > 0) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```
Comment (benj56) [inline src/login/loginEpic.ts:66]: 🚔

File: src/login/loginEpic.ts [hunk 3]
```diff
@@ -61,10 +62,12 @@ const rehydrateEpic = (action$: Observable<Action>, state$: StateObservable<Root
   return action$.pipe(
     filter((action) => action.type === 'persist/REHYDRATE'),
     tap((_action) => {
-      if ((state$.value.core.login.account?.length ?? 0) > 0) {
-        timeTacApi.setConfig({ account: state$.value.core.login.account });
+      if (state$.value.core.login.account !== undefined && state$.value.core.login.account.length > 0) {
```
Comment (benj56) [inline src/login/loginEpic.ts:65]:
```suggestion
      if (state$.value.core.login.account != null && state$.value.core.login.account.length > 0) {
```

File: src/login/loginEpic.ts [hunk 4]
```diff
@@ -91,8 +94,11 @@ const receiveSSOCodeEpic = (action$: Observable<Action>, state$: StateObservable
       return from(
         timeTacApi.authentication.login({
           grant_type: 'authorization_code',
-          client_id: action.payload.clientId ?? timeTacApi.config.clientId ?? '',
-          client_secret: action.payload.clientSecret ?? timeTacApi.config.clientSecret ?? '',
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```
Comment (benj56) [inline src/login/loginEpic.ts:97]: why are the ignores needed now?

**Review by benj56 — APPROVED** [review]

**Review by tomi-timetac — APPROVED** [review]

**Pull request merged**